apps/tk-capture-book/features/book/components/my-books.tsx

```
@@ -87,74 +86,75 @@ export default function HomeScreen() {
-          <View className="mb-4 h-12 w-12 items-center justify-center rounded-full bg-destructive/10">
-            <Text className="text-2xl">⚠️</Text>
-          </View>
-          <Text variant="title" size="lg" className="mb-2 text-center">
-            {t("common.error.title")}
-          </Text>
-          <Text variant="muted" className="text-center">
-            {error instanceof Error ? error.message : t("common.error.message")}
-          </Text>
-        </CardContent>
-      </Card>
+  const Error = () => (
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)